src/release/main.d

```diff
@@ -183,6 +183,20 @@ void main ( string[] params )
         }
     }
 
+    auto email_body = craftMail(con, repo, myrelease.type,
+        list.actions
+            .map!(a=>cast(ReleaseAction) a)
+            .filter!(a=>a !is null)
+            .array);
+
+    writefln("This is the announcement email:\n-----\n%s\n-----", email_body);
+
+    if (readYesNoResponse("Would you like to send it to <%s> now?",
```

Comment:
Why `<%s>`? Normally e-mail recipients are written like `First Last <mail@example.com>`, so having that surrounded by `<>` would be weird. I would just use `%s`.